.travis.yml

```diff
@@ -1,24 +1,59 @@
 ---
-language: python
+sudo: required
+cache: pip
 
 env:
   global:
     - HELM_URL=https://storage.googleapis.com/kubernetes-helm
     - HELM_TGZ=helm-v2.14.1-linux-amd64.tar.gz
+    - CHANGE_MINIKUBE_NONE_USER=true
+    - K8S_VERSION="v1.13.1"
+    - MINIKUBE_VERSION="v1.7.3"
 
 install:
   # Installing Helm
   - wget -q ${HELM_URL}/${HELM_TGZ}
   - tar xzfv ${HELM_TGZ}
   - PATH=`pwd`/linux-amd64/:$PATH
-  - helm init --client-only
+
+before_script:
+  - sudo mount --make-rshared /
+  - sudo apt-get install -y socat
```

Comment:
I don't see socat being called anywhere. Is this a minikube dependency?

Reply:
well, I haven't tried it now, but it used to be due to an issue between helm and minikube. Not sure on current versions.